tests/contrib/test_tensorflow.py

```diff
@@ -0,0 +1,221 @@
+import os
+
+import numpy as np
+import pytest
+import tensorflow as tf
+from pydantic import parse_obj_as
+
+from mlem.constants import PREDICT_METHOD_NAME
+from mlem.core.artifacts import LOCAL_STORAGE
+from mlem.core.data_type import DataAnalyzer
+from mlem.core.errors import DeserializationError, SerializationError
+from mlem.core.model import ModelAnalyzer
+
+
+@pytest.fixture
+def np_data():
+    return np.random.random((100, 32))
+
+
+@pytest.fixture
+def tensor_data(np_data):
+    return tf.convert_to_tensor(np_data, dtype=tf.float32)
+
+
+@pytest.fixture
+def tftt(tensor_data):
+    return DataAnalyzer.analyze(tensor_data)
+
+
+@pytest.fixture
+def tftt_3d(tensor_data):
+    return DataAnalyzer.analyze(
+        tf.tile(tf.expand_dims(tensor_data, -1), [1, 1, 20])
+    )
+
+
+@pytest.mark.skipif(
```

Comment:
not needed since we got correct env in tests

Reply:
thanks for removing